mod.conf

```diff
@@ -1 +1,4 @@
 name = hangglider
+depends = default, wool
+optional_depends = minetest_systemd, areas, player_monoids
+min_minetest_version = 5.0.0
```

Comment:
Patch version number is not documented (and should not be needed for compatibility with semantic versioning), only major and minor `5.0`.
I think it works anyway...